src/zappy_gui/GUI.hpp

```diff
@@ -0,0 +1,46 @@
+/*
+** EPITECH PROJECT, 2022
+** ZAPPYMIRROR
+** File description:
+** GUI
+*/
+
+/// \file GUI class
```

Comment:
\file must be the path of the file

Use a \brief if you want to describe the file